website/templates/emails/welcome_osf4i.html.mako

```diff
@@ -11,37 +11,50 @@
 
 Hello ${user.fullname},<br>
 <br>
-Thank you for verifying your OSF account, a free, open source service maintained by the Center for Open Science. Here are a few things you can do with the OSF:<br>
+Thank you for verifying your on OSF, a free, open source service maintained by the Center for Open Science. Here are a few things you can do with the OSF:<br>
```

Comment:
"Thank you for verifying your on OSF"
should be 
"Thank you for verifying your account on OSF"

Reply:
"Here are a few things you can do with the OSF"
should be 
"Here are a few things you can do with OSF"